src/components/Forms/TextFieldHint.js

```diff
@@ -35,17 +37,22 @@ class TextFieldHint extends Component {
     show: PropTypes.bool,
     /** Override styles */
     style: PropTypes.object,
+    /** A uniq id */
+    inputId: PropTypes.string,
   }
 
   static defaultProps = {
     disabled: false,
   }
 
   render() {
-    const { disabled, show, style, text } = this.props
+    const { disabled, show, style, text, inputId } = this.props
 
     return (
-      <div style={[styles.root, style, disabled && styles.disabled, show && styles.show]}>
+      <div
+        id={`hint_${inputId}`}
```

Comment:
It might be better for future reusability to apply the id prop directly here, and pass in `hint_${inputId}` from the parent component.